api/v1alpha2/compatibility_mode.go

```diff
@@ -9,7 +9,6 @@ import (
 // should be updated with every Istio bump according to the release notes
 // current env comes from: Istio 1.21, compatibilityVersion 1.20
 var pilotCompatibilityEnvVars = map[string]string{
-	"ENABLE_EXTERNAL_NAME_ALIAS":                                       "false",
```

Comment:
why we do not apply this env var with the compatibilty mode ?

Reply:
because of the `istio-operator.kyma-project.io/disable-external-name-alias` annotation we spoke yesterday in the meeting, that is influencing this env var presence

Reply:
because of the `istio-operator.kyma-project.io/disable-external-name-alias` annotation , that is influencing this env var presence

Reply:
we want to move to compatilityMode only, why there is a problem having temporarily both ? we want to eventually get rid of the annotation

Reply:
we discussed it yesterday in the meeting, let's sync